### ISSUE TITLE
↔ [Converter] Add support for Tensor.scatter_add_ in TRTorch

### ISSUE THREAD

**Issue by Linux-cpp-lisp** [NVIDIA/Torch-TensorRT#546]
##

- **Function Schema**: `scatter_add_fn = void(*)(Tensor& self, int64_t dim, const Tensor& index, const Tensor& src);` (https://github.com/pytorch/pytorch/blob/14badd9929e3fb6aad74983c6ba82658be6ab109/aten/src/ATen/native/TensorAdvancedIndexing.h)

- **Original PyTorch API**: Tensor.scatter_add_(dim, index, src) → Tensor

- **Relevant TensorRT Documentation**: n/a

## Alternatives

This can be inefficiently implemented as (psuedocode):
```
for idx in range(self.shape[dim]):
    self[..., idx, ...] += src[index == idx].sum(dim=dim)
```

## Additional context

This operation is very useful for graph neural networks, allowing the implementation of graph convolution steps where values computed on edges are accumulated into per-node features using the central node of the edge as the scattering index.

**Comment by Daniel-Kelvich** [comment]
Any updates on this feature?

**Comment by Linux-cpp-lisp** [comment]
Same question to @chengyute, just whether this is something that the TRTorch team is interested in adding eventually.

Thanks!
